src/mame/drivers/goldstar.cpp

```diff
@@ -17671,6 +17712,38 @@ void wingco_state::init_luckylad()
 	}
 }
 
+void cmaster_state::init_chthree()
+{
+	// Address swapping in 0x10 bytes blocks.
+	// 01234567-89abcdef <> c9ba8dfe - 41320576
+ 
+	uint8_t *ROM = memregion("maincpu")->base();
+	uint8_t row; 
+	uint16_t addr;
+
+	for (int A = 0000; A < 0x8000; A++)
```

Comment:
Please use `snake_case` for local variables, following the conventions used throughout the codebase.

Reply:
There are a lot of lines using the same "A" variable into this driver.....must I edit all the wrong lines?

Reply:
You can change them all now if you want or leave that for another time – I just don't want any more added.  We use `SCREAMING_SNAKE_CASE` for enumerated constants, preprocessor macros and certain globals that behave like constants (mostly `constexpr` globals and device types), `LlamaCase` for template parameters, and `snake_case` for pretty much everything else.

Reply:
ok.